test/cookbooks/app_test/recipes/app3_postrun.rb

```diff
@@ -8,7 +8,10 @@
       times += 1
       sleep 5
       puts "Still waiting for code.mulgara.org to start... #{times * 5} seconds"
-      raise('Failed to start code.mulgara.org') if times > 30
+      if times > 30
+        puts '! Failed to start code.mulgara.org, could just be first run !'
+        break
+      end
```

Comment:
Is there no other way around this other than doing a second run?

Reply:
@daviddjh ping?

Reply:
@ramereth I think the second run is necessary for the `edit_resource!`s immediately after this block of code, and for other resources in the suite that aren't idempotent. However, now I'm not sure why this block of code is necessary in the first place. We have an inspec test for a connection to `code.mulgara.org` that passes without this "wait_for_mulgara" block being present.